### ISSUE TITLE
Missing .json file, or user error

### ISSUE THREAD

**Issue by AJKimberly** [cSquaerd/CursaTetra#6]
I'm using Linux - an Ubuntu derivitave called Pop!_OS. I used my Terminal (Bash 5.0.17) to clone the repository, navigated into the directory, then typed:
>>> python3 cursatetra.py
It appears to load the board but I get the following error code (I attempted to format it, spacing is crazy on the Terminal):

Traceback (most recent call last): File "cursatetra.py", line 123, in ctMain scoreFile = open(scoreFileName, 'r')
FileNotFoundError: [Errno 2] No such file or directory: '.ctHiScrs.json'

During handling of the above exception, another exception occurred:
Traceback (most recent call last): File "cursatetra.py", line 690, in <module> ctMain()
File "cursatetra.py", line 137, in ctMain scoreFile = open(scoreFileName, 'w')
PermissionError: [Errno 13] Permission denied: '.ctHiScrs.json'

If you point me in the right direction, I'd love to try to sort this out but I don't see any .json files in the repo. Should I remove references to it? Should I do something else?

**Comment by cSquaerd** [comment]
Do you have write access in the directory you cloned the repo into? Normally, the high-score file is hidden (with a leading .) and created by the main program if it isn't detected. Once created, it is then read and written to, to keep your high scores logged.

The error on 137 is from the program trying to write the file into your directory, and failing since the OS tells it it doesn't have permission. It's "another exception" since I use a try statement to look for the first error, and handle it.

In short, check your user permissions on the directory. You must be able to create and edit files.

[Edit] If you run `ls -la` you need to see something similar to below. Note the `rw-` on most files and `drwx` on the `./`, meaning this directory:

4.0K drwxr-xr-x  2 charlie charlie 4.0K Feb  6 17:13 __pycache__/
4.0K drwxr-xr-x  4 charlie charlie 4.0K Feb  6 17:30 ./
4.0K drwxr-xr-x 24 charlie charlie 4.0K Jan  7 02:12 ../
4.0K drwxr-xr-x  8 charlie charlie 4.0K Feb  6 17:30 .git/
4.0K -rw-r--r--  1 charlie charlie  713 Jan  9 19:43 .ctHiScrs.json
8.0K -rw-r--r--  1 charlie charlie 5.3K Apr 29  2019 oldrotate.md
8.0K -rw-r--r--  1 charlie charlie 5.2K May 15  2020 README.md
1.8M -rw-r--r--  1 charlie charlie 1.8M Apr 29  2019 readmeBanner.png
4.0K -rw-r--r--  1 charlie charlie 1.8K Dec 18  2019 ct_cell.py
8.0K -rw-r--r--  1 charlie charlie 4.8K Feb  6 00:56 ct_draw.py
4.0K -rw-r--r--  1 charlie charlie 1.7K Jun  5  2019 ct_label.py
 24K -rw-r--r--  1 charlie charlie  22K Dec 20  2019 cursatetra.py
 28K -rw-r--r--  1 charlie charlie  27K Feb  6 17:13 Piece.py

Alternately, you can carefully comment out all lines dealing with the high score if you cannot gain access, though I would not recommend going this far.

**Comment by AJKimberly** [comment]
This worked great, thank you! I was able to use the 'chown' command to change my 'user' and 'group' as they were both listed as root instead of my username. For anybody else who encounters this problem my solution looked like this in the Terminal: 

`/CursaTetra$ sudo chown akimberly: ct_cell.py`

I then repeated for each file in the directory. I could've just listed all the filenames at the end and simply separated them by spaces.

Thanks for your help on it, the game is a ton of fun!
